app/utilities/drawing/master.py

```diff
@@ -0,0 +1,161 @@
+import sys
+import os
+import PIL
+from PIL import Image, ImageDraw, ImageOps, ImageFont, ImageEnhance, ImageFilter
```

Comment:
Why are you importing PIL, but then importing parts of PIL?

```suggestion

from PIL import Image, ImageDraw, ImageOps, ImageFont, ImageEnhance, ImageFilter```